src/Tizen.Uix.Sticker/Tizen.Uix.Sticker/StickerProvider.cs

```diff
@@ -0,0 +1,297 @@
+                    throw ExceptionFactory.CreateException(ErrorCode.OperationFailed);
+                else
+                    throw ExceptionFactory.CreateException(error);
+            }
+        }
+
+        /// <summary>
+        /// Inserts a sticker data to the sticker database.
+        /// </summary>
+        /// <since_tizen> 10 </since_tizen>
+        /// <feature>http://tizen.org/feature/ui_service.sticker</feature>
+        /// <param name="data">The sticker data to be saved.</param>
+        /// <exception cref="NotSupportedException">The required feature is not supported.</exception>
+        /// <exception cref="ArgumentException">This exception can be due to an invalid parameter.</exception>
+        /// <exception cref="InvalidOperationException">
+        /// This can occur due to the following reasons:
+        /// 1) This exception can be due to operation failed.
+        /// 2) This exception can be due to file exists.
+        /// 3) This exception can be due to no such file.
+        /// </exception>
+        public static void InsertData(StickerData data)
+        {
+            ErrorCode error = StickerProviderInsertData(_handle, data.SafeStickerDataHandle);
+            if (error != ErrorCode.None)
+            {
+                Log.Error(LogTag, "InsertData Failed with error " + error);
+                throw ExceptionFactory.CreateException(error);
+            }
+        }
+
+        /// <summary>
+        /// Inserts a sticker data using JSON file.
+        /// </summary>
+        /// <since_tizen> 10 </since_tizen>
+        /// <feature>http://tizen.org/feature/ui_service.sticker</feature>
+        /// <remarks>
+        /// All data except thumbnail, description, and display_type must be set in the JSON file to insert the sticker data.
+        /// @a jsonPath must have a non-null value and must be an existing file.
+        /// If the URI type is lacal path, the sticker file is copied to a sticker directory.
+        /// It is recommended to delete your sticker files after inserting a sticker data.
+        /// </remarks>
+        /// <param name="jsonPath">The path of JSON file containing sticker data to be saved</param>
+        /// <exception cref="NotSupportedException">The required feature is not supported.</exception>
+        /// <exception cref="ArgumentException">This exception can be due to an invalid parameter.</exception>
+        /// <exception cref="InvalidOperationException">This exception can be due to operation failed.</exception>
+        public static Task InsertData(string jsonPath)
+        {
+            var task = new TaskCompletionSource<bool>();
+            _insertDelegate = (ErrorCode errorCode, IntPtr userData) =>
+            {
+                if (errorCode != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Exception occurred while inserting");
+                    task.SetException(ExceptionFactory.CreateException(errorCode));
+                }
```

Comment:
task완료를 하지 않는것 같습니다.
```suggestion
                }
                else
                {
                    task.SetResult(true);
                }
```

Reply:
추가했습니다.